crates/cxx-qt-gen/test_outputs/invokables.rs

```diff
@@ -43,56 +43,67 @@ mod ffi {
     extern "Rust" {
         #[doc(hidden)]
         #[cxx_name = "cppMethodWrapper"]
+        #[namespace = "cxx_qt::my_object"]
```

Comment:
Hm, does this actually have any effect?
The QObject itself is already associated with a namespace. As these are member functions, they can't have a namespace of their own, so does this change anything?

Reply:
Not actually sure, I just replaced all the TODOs, assuming whoever left them knew whether it was necessary, might need to check over some of them

Reply:
@ahayzen-kdab Do you have any idea of why we had that in there?

Reply:
Was this historical when we used to have free methods and before they were using `self` due to other reasons?

(eg we used to have a free method like `my_invokable(this, ...args)` instead of `T::my_invokable(..args)` )

As yes I think this isn't required anymore and instead the comments could be removed?

Reply:
Done, and replaced Todos with comments indicating namespace shouldn't be added